keyboards/era/sirind/tomak/readme.md

```diff
@@ -0,0 +1,27 @@
+# Tomak
+
+![Tomak](https://i.imgur.com/CmVR0G1.jpeg)
```

Comment:
```suggestion
![Tomak](https://i.imgur.com/CmVR0G1h.jpeg)
```

Reply:
Thanks for your help.
However, the modified URL address appears to be invalid.
Could you tell me why this should be changed?

Reply:
the original is also invalid